### DISCUSSION TITLE
Unable to configure SSL for Postal

### DISCUSSION THREAD

**Discussion by huynhphuong10284** [postalserver/postal#2055] [Help with using Postal]
As I followed instructions from https://github.com/postalserver/postal/issues/481
and https://github.com/postalserver/postal/issues/1033, but I cannot found "webroot-path /opt/postal/app/public". From my server, it showed:

root@mail:~# cd  /opt/postal/
root@mail:/opt/postal# ls
caddy-data  config  install

If I try to run "certbot certonly --webroot --webroot-path /opt/postal/app/public --domains postal.example.com", an error says "/opt/postal/app/public does not exist or is not a directory".

Could you please review and help me on this case? I'm using the latest version and follow installation guide from https://docs.postalserver.io/install/installation.

**Comment by willpower232** [comment]
If you're using caddy, you won't need to use certbot. I'm not sure where the caddy certificates are kept as files off the top of my head but it has been discussed more than a few times around here.

**Reply by huynhphuong10284** [reply]
Thank you for your quick answer, could you please instruct me how to do this? As I cannot find reference document to follow

**Reply by jazer143** [reply]
There's no configuration needed, if you server has internet connection, "Caddy will issue SSL certificate for your domain and you'll be able to immediately access the Postal web interface"

**Reply by huynhphuong10284** [reply]
I've tried many times by following the installation instructions https://docs.postalserver.io/install/installation/. However, after finished - I cannot launch Portal web interface via https. I have to launch http instead of, for example http://IP or domain:5000. Could you please review and help me on this?

**Reply by willpower232** [reply]
There is an example Caddyfile in the install repo: https://github.com/postalserver/install/blob/main/examples/Caddyfile

You could use `netstat -lnp` to confirm that Caddy is running your ports 80 and 443.

**Reply by huynhphuong10284** [reply]
Thanks all for supports, I've tried with a new fresh server and ssl works for my setup. The root cause is my old server installed wordpress with ssl enabled and its' conflicted port. So, let's me concern how can we use Caddy while other service is using port =443?

**Reply by willpower232** [reply]
You're probably using apache or nginx to run wordpress so you could either configure that to point at your wordpress OR you could configure caddy to deal with your wordpress site and turn off apache or nginx.

**Reply by huynhphuong10284** [reply]
Thanks, it works
